packages/terra-avatar/src/index.js

```diff
@@ -0,0 +1,12 @@
+import Avatar from './variants/Avatar';
+import Facility from './variants/Facility';
+import SharedUser from './variants/SharedUser';
+
+const Variants = {
+  Avatar,
+  Facility,
+  SharedUser,
+};
+
+export default Variants;
```

Comment:
Are these changes a MVB? Also, should we be defining Avatar as the default?

Reply:
Yes, this is a MVB. I'm indifferent on that - anyone else have opinions?

Reply:
I think exporting Avatar as the default makes sense.

Reply:
@emilyrohrbough @tbiethman updated https://github.com/cerner/terra-core/pull/2061/commits/037512311485e141baf171804db0f971a700200b